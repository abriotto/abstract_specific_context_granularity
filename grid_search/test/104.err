  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:11,  8.18it/s]  2%|###7                                                                                                                                                                                        | 2/99 [00:00<00:12,  8.05it/s]  4%|#######5                                                                                                                                                                                    | 4/99 [00:00<00:07, 12.63it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:08, 11.49it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:00<00:06, 14.54it/s] 12%|######################6                                                                                                                                                                    | 12/99 [00:01<00:07, 11.52it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:01<00:07, 11.17it/s] 16%|##############################2                                                                                                                                                            | 16/99 [00:01<00:07, 10.99it/s] 19%|###################################8                                                                                                                                                       | 19/99 [00:01<00:05, 14.13it/s] 21%|#######################################6                                                                                                                                                   | 21/99 [00:01<00:07, 11.03it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:02<00:07, 10.10it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:08,  8.57it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:02<00:08,  8.50it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:02<00:07,  9.32it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:03<00:08,  8.16it/s] 33%|##############################################################3                                                                                                                            | 33/99 [00:03<00:06,  9.82it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:07,  8.12it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:03<00:08,  7.53it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:04<00:08,  7.15it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:04<00:07,  7.92it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:04<00:07,  7.68it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:07,  7.13it/s] 43%|#################################################################################2                                                                                                         | 43/99 [00:04<00:07,  7.01it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:04<00:06,  8.53it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:05<00:06,  8.38it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:05<00:04, 10.28it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:05<00:05,  9.50it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:05<00:05,  8.15it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:05<00:06,  7.48it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:06<00:06,  7.18it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:06<00:04,  9.28it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:03, 10.42it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:06<00:03, 10.96it/s] 64%|#######################################################################################################################                                                                    | 63/99 [00:06<00:03, 10.02it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:07<00:04,  8.20it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:07<00:03,  8.28it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:07<00:03,  7.89it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:07<00:03,  9.05it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:07<00:02,  9.51it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:08<00:03,  8.54it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:08<00:03,  7.96it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:03,  7.52it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:02,  9.04it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:08<00:02,  8.04it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:08<00:02,  7.57it/s] 82%|#########################################################################################################################################################                                  | 81/99 [00:08<00:02,  8.85it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:09<00:02,  8.04it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:02,  7.42it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:02,  6.96it/s] 86%|################################################################################################################################################################5                          | 85/99 [00:09<00:02,  6.70it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:02,  6.47it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:09<00:01,  6.35it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:10<00:01,  7.10it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:10<00:01,  6.73it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:10<00:00,  7.16it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00,  6.85it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  9.07it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:11<00:00,  8.82it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  9.85it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.73it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  8%|###############                                                                                                                                                                             | 2/25 [00:00<00:02,  8.34it/s] 12%|######################5                                                                                                                                                                     | 3/25 [00:00<00:03,  7.19it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:01, 10.72it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:01, 11.42it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:01, 11.88it/s] 44%|##################################################################################2                                                                                                        | 11/25 [00:01<00:01,  8.96it/s] 52%|#################################################################################################2                                                                                         | 13/25 [00:01<00:01, 10.19it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:01<00:01,  8.29it/s] 64%|#######################################################################################################################6                                                                   | 16/25 [00:01<00:01,  7.77it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:01<00:00,  9.24it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00,  7.74it/s] 84%|#############################################################################################################################################################                              | 21/25 [00:02<00:00,  7.44it/s] 92%|############################################################################################################################################################################               | 23/25 [00:02<00:00,  8.70it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  8.17it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  9.00it/s]
