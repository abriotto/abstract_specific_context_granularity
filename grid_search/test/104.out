# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6954686641693115, "acc": 0.5199386477470398, "length": 3.316772937774658, "mode": "train", "epoch": 1}
{"loss": 0.6951380968093872, "acc": 0.5141791105270386, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6296192407608032, "acc": 0.6614476442337036, "length": 3.06878924369812, "mode": "train", "epoch": 2}
{"loss": 0.5796394348144531, "acc": 0.7042281627655029, "length": 3.572139263153076, "mode": "test", "epoch": 2}
{"loss": 0.5221918225288391, "acc": 0.7507065534591675, "length": 3.310131788253784, "mode": "train", "epoch": 3}
{"loss": 0.5081769824028015, "acc": 0.7623594999313354, "length": 3.427860736846924, "mode": "test", "epoch": 3}
{"loss": 0.4645642340183258, "acc": 0.8008805513381958, "length": 3.444173574447632, "mode": "train", "epoch": 4}
{"loss": 0.4680396020412445, "acc": 0.8081603050231934, "length": 3.4941956996917725, "mode": "test", "epoch": 4}
{"loss": 0.41550564765930176, "acc": 0.8201971054077148, "length": 3.5162270069122314, "mode": "train", "epoch": 5}
{"loss": 0.44028979539871216, "acc": 0.8059322834014893, "length": 3.507462739944458, "mode": "test", "epoch": 5}
{"loss": 0.37630870938301086, "acc": 0.8395959734916687, "length": 3.5136988162994385, "mode": "train", "epoch": 6}
{"loss": 0.4224426746368408, "acc": 0.8219626545906067, "length": 3.5704808235168457, "mode": "test", "epoch": 6}
{"loss": 0.3501119911670685, "acc": 0.8517658114433289, "length": 3.55418062210083, "mode": "train", "epoch": 7}
{"loss": 0.39556363224983215, "acc": 0.8301352262496948, "length": 3.5804312229156494, "mode": "test", "epoch": 7}
{"loss": 0.32721829414367676, "acc": 0.8611446619033813, "length": 3.6427793502807617, "mode": "train", "epoch": 8}
{"loss": 0.4071415960788727, "acc": 0.8349181413650513, "length": 3.694859027862549, "mode": "test", "epoch": 8}
{"loss": 0.3050253689289093, "acc": 0.8720420002937317, "length": 3.7510781288146973, "mode": "train", "epoch": 9}
{"loss": 0.3665941655635834, "acc": 0.8496279716491699, "length": 3.842454433441162, "mode": "test", "epoch": 9}
{"loss": 0.28890228271484375, "acc": 0.8786200284957886, "length": 3.873237371444702, "mode": "train", "epoch": 10}
{"loss": 0.33070871233940125, "acc": 0.8644986152648926, "length": 3.9718077182769775, "mode": "test", "epoch": 10}
{"loss": 0.2649306654930115, "acc": 0.8916419148445129, "length": 3.938420534133911, "mode": "train", "epoch": 11}
{"loss": 0.3327896296977997, "acc": 0.8644096255302429, "length": 3.9917080402374268, "mode": "test", "epoch": 11}
{"loss": 0.25216296315193176, "acc": 0.8980505466461182, "length": 3.9712820053100586, "mode": "train", "epoch": 12}
{"loss": 0.3308790326118469, "acc": 0.8739635944366455, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.24586482346057892, "acc": 0.9043439626693726, "length": 3.981614112854004, "mode": "train", "epoch": 13}
{"loss": 0.3275585174560547, "acc": 0.8708128929138184, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.23457276821136475, "acc": 0.9082891345024109, "length": 3.9860000610351562, "mode": "train", "epoch": 14}
{"loss": 0.3219992518424988, "acc": 0.8813431859016418, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.2259211540222168, "acc": 0.9112728238105774, "length": 3.991011142730713, "mode": "train", "epoch": 15}
{"loss": 0.2983803451061249, "acc": 0.8847430348396301, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.22708760201931, "acc": 0.9140620231628418, "length": 3.9928438663482666, "mode": "train", "epoch": 16}
{"loss": 0.2969140112400055, "acc": 0.8805140256881714, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.20927700400352478, "acc": 0.9212634563446045, "length": 3.9941465854644775, "mode": "train", "epoch": 17}
{"loss": 0.2802450358867645, "acc": 0.8892203569412231, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.20299874246120453, "acc": 0.9243302345275879, "length": 3.995011568069458, "mode": "train", "epoch": 18}
{"loss": 0.2650931775569916, "acc": 0.9014924764633179, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.19150754809379578, "acc": 0.9284927248954773, "length": 3.9971394538879395, "mode": "train", "epoch": 19}
{"loss": 0.28928348422050476, "acc": 0.8951076865196228, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.1922440081834793, "acc": 0.9268438816070557, "length": 3.997634172439575, "mode": "train", "epoch": 20}
{"loss": 0.28613191843032837, "acc": 0.8968489766120911, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.19298136234283447, "acc": 0.9290726780891418, "length": 3.9981706142425537, "mode": "train", "epoch": 21}
{"loss": 0.2535806894302368, "acc": 0.9075455069541931, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.18967103958129883, "acc": 0.9296144247055054, "length": 3.9978325366973877, "mode": "train", "epoch": 22}
{"loss": 0.26791536808013916, "acc": 0.8902155756950378, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.1823517233133316, "acc": 0.931881844997406, "length": 3.997523546218872, "mode": "train", "epoch": 23}
{"loss": 0.24847529828548431, "acc": 0.9045604467391968, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.16831423342227936, "acc": 0.9384705424308777, "length": 3.998563289642334, "mode": "train", "epoch": 24}
{"loss": 0.2501726746559143, "acc": 0.9115258455276489, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.1693992018699646, "acc": 0.937618613243103, "length": 3.9983928203582764, "mode": "train", "epoch": 25}
{"loss": 0.2300642728805542, "acc": 0.9189056158065796, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.1651596873998642, "acc": 0.9398379325866699, "length": 3.9992012977600098, "mode": "train", "epoch": 26}
{"loss": 0.22806434333324432, "acc": 0.9196517467498779, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.15901689231395721, "acc": 0.9433741569519043, "length": 3.9991464614868164, "mode": "train", "epoch": 27}
{"loss": 0.2176337093114853, "acc": 0.9252071380615234, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.15405476093292236, "acc": 0.9461141228675842, "length": 3.9992172718048096, "mode": "train", "epoch": 28}
{"loss": 0.21258237957954407, "acc": 0.9285239577293396, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.16163519024848938, "acc": 0.9403181672096252, "length": 3.999389886856079, "mode": "train", "epoch": 29}
{"loss": 0.2222447246313095, "acc": 0.9199004173278809, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.1770574003458023, "acc": 0.9340800642967224, "length": 3.9975457191467285, "mode": "train", "epoch": 30}
{"loss": 0.25493472814559937, "acc": 0.9202321171760559, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.17151296138763428, "acc": 0.938095211982727, "length": 3.9998538494110107, "mode": "train", "epoch": 31}
{"loss": 0.22159810364246368, "acc": 0.9255390167236328, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.1672692447900772, "acc": 0.9403841495513916, "length": 3.9996352195739746, "mode": "train", "epoch": 32}
{"loss": 0.22009027004241943, "acc": 0.9276949167251587, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.14318449795246124, "acc": 0.9488807916641235, "length": 3.999796152114868, "mode": "train", "epoch": 33}
{"loss": 0.20570491254329681, "acc": 0.9338307976722717, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.14065976440906525, "acc": 0.9505150318145752, "length": 3.9998083114624023, "mode": "train", "epoch": 34}
{"loss": 0.2136005461215973, "acc": 0.9376451373100281, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.1355758160352707, "acc": 0.9532890915870667, "length": 3.9999122619628906, "mode": "train", "epoch": 35}
{"loss": 0.1990959793329239, "acc": 0.9392206072807312, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.1354781985282898, "acc": 0.9527875781059265, "length": 3.999882936477661, "mode": "train", "epoch": 36}
{"loss": 0.1956278532743454, "acc": 0.9395522475242615, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.13316823542118073, "acc": 0.9518731832504272, "length": 3.9998695850372314, "mode": "train", "epoch": 37}
{"loss": 0.19960957765579224, "acc": 0.9342452883720398, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.1384204924106598, "acc": 0.9516420364379883, "length": 3.999758720397949, "mode": "train", "epoch": 38}
{"loss": 0.19346974790096283, "acc": 0.9399670362472534, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.13308489322662354, "acc": 0.9537613987922668, "length": 3.999967575073242, "mode": "train", "epoch": 39}
{"loss": 0.18841424584388733, "acc": 0.9349088072776794, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.12690383195877075, "acc": 0.9569120407104492, "length": 3.9998939037323, "mode": "train", "epoch": 40}
{"loss": 0.1945272982120514, "acc": 0.9372304677963257, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.12784743309020996, "acc": 0.9544198513031006, "length": 3.9998772144317627, "mode": "train", "epoch": 41}
{"loss": 0.19369778037071228, "acc": 0.9393863081932068, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.12547087669372559, "acc": 0.9570274949073792, "length": 3.9999895095825195, "mode": "train", "epoch": 42}
{"loss": 0.20044870674610138, "acc": 0.9399667382240295, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.12460103631019592, "acc": 0.9563637375831604, "length": 3.999993085861206, "mode": "train", "epoch": 43}
{"loss": 0.20564129948616028, "acc": 0.9391376376152039, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.11628924310207367, "acc": 0.9576364755630493, "length": 3.9999918937683105, "mode": "train", "epoch": 44}
{"loss": 0.19923661649227142, "acc": 0.9417082667350769, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.11889606714248657, "acc": 0.9578301310539246, "length": 3.9999938011169434, "mode": "train", "epoch": 45}
{"loss": 0.1875796765089035, "acc": 0.9419568777084351, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.12770099937915802, "acc": 0.9550623893737793, "length": 3.999983072280884, "mode": "train", "epoch": 46}
{"loss": 0.2048056572675705, "acc": 0.9425374865531921, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.14695791900157928, "acc": 0.9520469903945923, "length": 3.999990224838257, "mode": "train", "epoch": 47}
{"loss": 0.2070385068655014, "acc": 0.9446931481361389, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.13030482828617096, "acc": 0.9561944007873535, "length": 3.99994158744812, "mode": "train", "epoch": 48}
{"loss": 0.2141347974538803, "acc": 0.9370647668838501, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.13028332591056824, "acc": 0.9519919753074646, "length": 3.999997615814209, "mode": "train", "epoch": 49}
{"loss": 0.22232425212860107, "acc": 0.9276120066642761, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.13206978142261505, "acc": 0.9535959362983704, "length": 3.9999842643737793, "mode": "train", "epoch": 50}
{"loss": 0.20723171532154083, "acc": 0.9412106871604919, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.12836463749408722, "acc": 0.9571892619132996, "length": 3.999361991882324, "mode": "train", "epoch": 51}
{"loss": 0.21565234661102295, "acc": 0.936650276184082, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.12170307338237762, "acc": 0.9553122520446777, "length": 3.9999935626983643, "mode": "train", "epoch": 52}
{"loss": 0.21353058516979218, "acc": 0.9376451373100281, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.17453278601169586, "acc": 0.9422799944877625, "length": 3.999999523162842, "mode": "train", "epoch": 53}
{"loss": 0.21231785416603088, "acc": 0.9345771670341492, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.1777525246143341, "acc": 0.9395681023597717, "length": 3.99999737739563, "mode": "train", "epoch": 54}
{"loss": 0.2109833061695099, "acc": 0.9318409562110901, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.1520586609840393, "acc": 0.9445486068725586, "length": 3.999990701675415, "mode": "train", "epoch": 55}
{"loss": 0.19895492494106293, "acc": 0.9331673979759216, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.14020605385303497, "acc": 0.9490867257118225, "length": 3.999997615814209, "mode": "train", "epoch": 56}
{"loss": 0.1993853896856308, "acc": 0.9338310360908508, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.12374264001846313, "acc": 0.9555613994598389, "length": 3.999995231628418, "mode": "train", "epoch": 57}
{"loss": 0.17461036145687103, "acc": 0.9421226978302002, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.12521052360534668, "acc": 0.9554229974746704, "length": 3.9999938011169434, "mode": "train", "epoch": 58}
{"loss": 0.1838436871767044, "acc": 0.9387231469154358, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.11714574694633484, "acc": 0.9594355225563049, "length": 3.999997615814209, "mode": "train", "epoch": 59}
{"loss": 0.1833392232656479, "acc": 0.9395523071289062, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.1143144890666008, "acc": 0.9603611826896667, "length": 3.9997870922088623, "mode": "train", "epoch": 60}
{"loss": 0.1852462887763977, "acc": 0.9440298676490784, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.10959245264530182, "acc": 0.9612886309623718, "length": 3.9999921321868896, "mode": "train", "epoch": 61}
{"loss": 0.18824785947799683, "acc": 0.9450247287750244, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.11394975334405899, "acc": 0.958743691444397, "length": 3.9999983310699463, "mode": "train", "epoch": 62}
{"loss": 0.18708311021327972, "acc": 0.9432836771011353, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.11071565002202988, "acc": 0.9598501324653625, "length": 3.9999897480010986, "mode": "train", "epoch": 63}
{"loss": 0.16822485625743866, "acc": 0.9452736377716064, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.11063125729560852, "acc": 0.9599889516830444, "length": 3.9999992847442627, "mode": "train", "epoch": 64}
{"loss": 0.17522086203098297, "acc": 0.9485904574394226, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.10983693599700928, "acc": 0.961344301700592, "length": 3.9999938011169434, "mode": "train", "epoch": 65}
{"loss": 0.17417703568935394, "acc": 0.9490879774093628, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.10906488448381424, "acc": 0.9621195197105408, "length": 3.9999983310699463, "mode": "train", "epoch": 66}
{"loss": 0.2007482796907425, "acc": 0.9403817057609558, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.10936132818460464, "acc": 0.9617874622344971, "length": 3.9999983310699463, "mode": "train", "epoch": 67}
{"loss": 0.17819546163082123, "acc": 0.9452736377716064, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.1054423451423645, "acc": 0.9612894058227539, "length": 3.9999988079071045, "mode": "train", "epoch": 68}
{"loss": 0.18325912952423096, "acc": 0.9459370374679565, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.09399770945310593, "acc": 0.9669339656829834, "length": 3.99999737739563, "mode": "train", "epoch": 69}
{"loss": 0.18449515104293823, "acc": 0.9442786574363708, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.09725309163331985, "acc": 0.9633646607398987, "length": 3.999999523162842, "mode": "train", "epoch": 70}
{"loss": 0.1726670116186142, "acc": 0.946766197681427, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.0998397022485733, "acc": 0.9657441973686218, "length": 3.999997615814209, "mode": "train", "epoch": 71}
{"loss": 0.18757256865501404, "acc": 0.9481757879257202, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.10145311802625656, "acc": 0.9637237191200256, "length": 3.999983549118042, "mode": "train", "epoch": 72}
{"loss": 0.2006693184375763, "acc": 0.9463517069816589, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.11068814992904663, "acc": 0.9620364308357239, "length": 4.0, "mode": "train", "epoch": 73}
{"loss": 0.20136845111846924, "acc": 0.9455224275588989, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.10119454562664032, "acc": 0.9655501246452332, "length": 3.999992609024048, "mode": "train", "epoch": 74}
{"loss": 0.18355561792850494, "acc": 0.9505804777145386, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.0953734889626503, "acc": 0.9652186632156372, "length": 3.9999988079071045, "mode": "train", "epoch": 75}
{"loss": 0.18448808789253235, "acc": 0.9465172290802002, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.10374028980731964, "acc": 0.9642500281333923, "length": 3.999997854232788, "mode": "train", "epoch": 76}
{"loss": 0.18078462779521942, "acc": 0.9475122094154358, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.09090661257505417, "acc": 0.9669063091278076, "length": 3.99999737739563, "mode": "train", "epoch": 77}
{"loss": 0.1798672080039978, "acc": 0.9513267278671265, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.11061543226242065, "acc": 0.9614830017089844, "length": 3.9999961853027344, "mode": "train", "epoch": 78}
{"loss": 0.22653046250343323, "acc": 0.9381425380706787, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.11478164047002792, "acc": 0.958493709564209, "length": 3.9999773502349854, "mode": "train", "epoch": 79}
{"loss": 0.19854971766471863, "acc": 0.9448590278625488, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.11219125241041183, "acc": 0.9601581692695618, "length": 3.9994542598724365, "mode": "train", "epoch": 80}
{"loss": 0.17261642217636108, "acc": 0.9490879774093628, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.09713641554117203, "acc": 0.9655965566635132, "length": 3.999847412109375, "mode": "train", "epoch": 81}
{"loss": 0.18095657229423523, "acc": 0.9479269981384277, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.11908479034900665, "acc": 0.9565300941467285, "length": 3.9999985694885254, "mode": "train", "epoch": 82}
{"loss": 0.21049903333187103, "acc": 0.9383082389831543, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.1182139441370964, "acc": 0.9591547846794128, "length": 3.9998600482940674, "mode": "train", "epoch": 83}
{"loss": 0.22485847771167755, "acc": 0.9396349191665649, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.09625431895256042, "acc": 0.9664599299430847, "length": 3.9998981952667236, "mode": "train", "epoch": 84}
{"loss": 0.19413882493972778, "acc": 0.9448591470718384, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.09670721739530563, "acc": 0.9654091000556946, "length": 3.9991931915283203, "mode": "train", "epoch": 85}
{"loss": 0.17927296459674835, "acc": 0.9518241286277771, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.1043938621878624, "acc": 0.9660487771034241, "length": 3.999999761581421, "mode": "train", "epoch": 86}
{"loss": 0.2093665897846222, "acc": 0.9429519772529602, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.10615725815296173, "acc": 0.9628664255142212, "length": 3.9999985694885254, "mode": "train", "epoch": 87}
{"loss": 0.20220422744750977, "acc": 0.9453566074371338, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.09850195795297623, "acc": 0.9655141830444336, "length": 3.999849319458008, "mode": "train", "epoch": 88}
{"loss": 0.18291917443275452, "acc": 0.9500829577445984, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.0891537070274353, "acc": 0.9668226838111877, "length": 3.999983787536621, "mode": "train", "epoch": 89}
{"loss": 0.18976585566997528, "acc": 0.9460198283195496, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.09058001637458801, "acc": 0.9677045941352844, "length": 3.9999492168426514, "mode": "train", "epoch": 90}
{"loss": 0.18473897874355316, "acc": 0.9517412185668945, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.0871061459183693, "acc": 0.9695835709571838, "length": 3.9994451999664307, "mode": "train", "epoch": 91}
{"loss": 0.20490097999572754, "acc": 0.9436152577400208, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.0831594318151474, "acc": 0.9719977974891663, "length": 3.999999523162842, "mode": "train", "epoch": 92}
{"loss": 0.2003733366727829, "acc": 0.9436980485916138, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.08712773770093918, "acc": 0.9697287082672119, "length": 3.9999983310699463, "mode": "train", "epoch": 93}
{"loss": 0.17961366474628448, "acc": 0.9506633281707764, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.08258049190044403, "acc": 0.971250593662262, "length": 3.999998092651367, "mode": "train", "epoch": 94}
{"loss": 0.17536742985248566, "acc": 0.9530681371688843, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.07859249413013458, "acc": 0.9727171659469604, "length": 3.999999523162842, "mode": "train", "epoch": 95}
{"loss": 0.1699550747871399, "acc": 0.9547262787818909, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.0822884738445282, "acc": 0.9704481959342957, "length": 3.9999983310699463, "mode": "train", "epoch": 96}
{"loss": 0.18954052031040192, "acc": 0.9514926671981812, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.07986988127231598, "acc": 0.9731045961380005, "length": 4.0, "mode": "train", "epoch": 97}
{"loss": 0.1736791431903839, "acc": 0.9556384682655334, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.08181492984294891, "acc": 0.9720251560211182, "length": 3.99999737739563, "mode": "train", "epoch": 98}
{"loss": 0.1651146411895752, "acc": 0.9527361989021301, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.08174078911542892, "acc": 0.9721354842185974, "length": 3.999990701675415, "mode": "train", "epoch": 99}
{"loss": 0.17569217085838318, "acc": 0.9523218274116516, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.08821970969438553, "acc": 0.9695903658866882, "length": 4.0, "mode": "train", "epoch": 100}
{"loss": 0.16016161441802979, "acc": 0.9573798775672913, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7403390407562256
